src/Rings/groebner.jl

```diff
@@ -512,7 +563,70 @@ function reduce(F::Vector{T}, G::Vector{T}; ordering::MonomialOrdering = default
 end
 
 @doc Markdown.doc"""
-        reduce_with_quotients_and_units(I::IdealGens, J::IdealGens; ordering::MonomialOrdering = default_ordering(base_ring(J)))
+	reduce_with_quotients_and_unit(g::T, F::Vector{T};
+          ordering::MonomialOrdering = default_ordering(parent(F[1]))) where {T <: MPolyElem}
+
+Return the unit, the quotients and the remainder in a weak standard representation for `g` on division by the polynomials in `F` with respect to `ordering`.
+
+	reduce_with_quotients(G::Vector{T}, F::Vector{T};
+          ordering::MonomialOrdering = default_ordering(parent(F[1]))) where {T <: MPolyElem}
+
+Return a `Vector` which contains, for each element `g` of `G`, a unit, quotients, and a remainder as above.
+
+!!! note
+    In the global case, a standard representation with a fully reduced remainder is computed.
+
+# Examples
+```jldoctest
+julia> R, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"]);
+
+julia> f1 = x^2+x^2*y; f2 = y^3+x*y*z; f3 = x^3*y^2+z^4;
+
+julia> g = x^3*y+x^5+x^2*y^2*z^2+z^6;
+
+julia> u, Q, h = reduce_with_quotients_and_unit(g, [f1,f2, f3], ordering = negdegrevlex(R))
+([x^3-x*y^2*z^2+x*y+y^2*z^2 0 y*z^2+z^2], 0, [y+1])
```

Comment:
Please generate this result again. It is still in the old ordering of the output.

Reply:
Fixed with some other doctests.